tests/ro/creator/test_roc_cli.py

```
@@ -0,0 +1,41 @@
+import os
+from pathlib import Path
+
+import pytest
+
+from sema.ro.creator.__main__ import _main
+
+MY_FOLDER = Path(__file__).parent
+ROCRATEROOT_FOLDER = MY_FOLDER / "data"
+
+
+def test_help(capfd):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_help`

Add return type annotation: `None`

(ANN201)